### ISSUE TITLE
Pokemon png/ sprite not working.

### ISSUE THREAD

**Issue by Rockydangerbuff** [felixphew/pokepaste#161]
Hi for some reason in pokemon showdown and pokepaste farfetch'd, farfetch'd galar form and sirfeched png/ sprites are not loading. Can you please add them in or tell me if there is any way I could do it myself thank you.

**Comment by felixphew** [comment]
Hi - could you give me a link to the pastes where they aren't loading? I'm guessing it's to do with the apostrophe (`'`) in their names. 

(Also, I don't control Showdown - you'll need to report that one to _their_ developers. But maybe wait until I fix this one first, as it could be something funny at your end.)

**Comment by Rockydangerbuff** [comment]
Sorry if my first comment was confusing, there is nothing wrong with showdown itself its just when I upload it to pokepaste the png dosen't show up here is the link.  

https://pokepast.es/9a01c43fb06367c1

**Comment by felixphew** [comment]
Yep, your pastes have a curly apostrophe (`’`) in them, rather than the straight one (`'`). I thought PokePaste still highlighted those fine, though... I'll look into it.

**Comment by SilverSight1** [comment]
Same issue, brought me to this thread here on github. :P edit: Yes manually editing the paste with a ' fixes the image.
